proposals/function-pointers.md

```diff
@@ -212,6 +212,25 @@ Restrictions of this feature:
   deliberately not specified by the language. This includes whether they are static vs. instance or
   exactly what signature they are emitted with.
 
+
+### Operators on Function Pointer Types
+
+The section in unsafe code on operators is modified as such:
+
+> In an unsafe context, several constructs are available for operating on all _pointer\_type_s but _funcptr\_type_s (except where noted):
```

Comment:
Suggest changing "but" to "that are not".  Better yet, because an exception to an exception is confusing, make the one bullet that permits function pointers a separate paragraph not included in this bullet list.